docs/api.rst

```diff
@@ -2085,6 +2085,24 @@ Certain utilities make working with async iterators easier, detailed below.
         :return: A list of every element in the async iterator.
         :rtype: list
 
+    .. method:: chunk(max_size)
+
+        Collects items into chunks of up to a given maximum size.
+        Another :class:`AsyncIterator` is returned which collects items into
+        :class:`list`\s of a given size. The maximum chunk size must be a positive integer.
+
```

Comment:
```suggestion
        .. versionadded:: 1.6
        
```

(or 2.0?)

Reply:
Omitted waiting on clarification from @Rapptz

Reply:
1.6 is fine